helpers/chains.ts

```diff
@@ -193,6 +193,9 @@ export enum CHAIN {
   BOUNCE_BIT = "bouncebit",
   ZIRCUIT = "zircuit",
   WC = "wc",
+  WORLD_CHAIN = "world-chain",
```

Comment:
line 195 is world chain, this is duplicate

Reply:
line 175 is BSQUARED

Reply:
I updated in the commit: 75a6c9374b6582a4d4c4db38705f6be1b161e41e